Cargo.toml

```diff
@@ -9,12 +9,15 @@ async-trait = "0.1"
 log = "0.4"
 serde = { version = "1.0", features = ["derive"] }
 serde_with = "1.6"
-web3 = { version = "0.13", default-features = false }
-primitive-types = { version = "0.7.3", features = ["fp-conversion"] }
+web3 = { version = "0.15", default-features = false, optional = true }
+primitive-types = { version = "0.8", features = ["fp-conversion"], optional = true }
+
+[features]
+web3_ = ["web3", "primitive-types"]
```

Comment:
Any reason there is an underscore here?

Reply:
Because the dependency is already called `web3` I cannot reuse the same name for the feature.

Reply:
I also find the _ a bit unpleasant. Maybe we can call it eth-node as this is the name of the module that would be added if the feature is enabled. But no big deal.

Reply:
The module is private, it is just the impl that gets added "silently" and it is normal to have dependencies as features. Usually the `web3 = ... optional` would be enough it's just that in case we also need primitive types.